fix(inferencer): `show code` button isn't visible on tutorials. (#4499)
refinedev/refine: .changeset/lucky-seals-yell.md

---
"@refinedev/inferencer": patch
---

> fix: show code button for inferencer isn't visible on tutorials.

Reviewer: ```suggestion
fix: show code button for inferencers was not visible in smaller screens
```
